decorators/trimmer_decorator.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    return unless @nameable.correct_name.length > 10
+
+    super[0..9]
+  end
```

Comment:
- [ ] Your trimmer returns `nil` if the name is less than or equal to 10 characters, but that is unexpected. We should simply return the capitalize version of the name, instead of returning `nil`. Kindly fix that.

  ![image](https://user-images.githubusercontent.com/54848363/230942420-82af04d5-776c-4688-8952-51a2ebbb30de.png)

  ```
  def correct_name
    super[0..9]
  end
  ```

Reply:
Hi @DammyShittu, thank you very mucho for the observation.